app/exporters/wikifactory.py

```diff
@@ -44,7 +44,7 @@ def wikifactory_api_request(
 ) -> Dict:
     headers = {"Authorization": f"Bearer {auth_token}"} if auth_token else None
     transport = RequestsHTTPTransport(
-        url=f"http://{settings.WIKIFACTORY_API_HOST}/api/graphql",
+        url=f"{settings.WIKIFACTORY_API_HOST}/api/graphql",
```

Comment:
Does this work with the default value of `WIKIFACTORY_API_HOST`? (which doesn't have the protocol)